.travis.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         - yarn install --frozen-lockfile
       script:
         - yarn format
+        - yarn test -- --coverage
```

Comment:
why the -- --coverage?

Reply:
it's to print the coverage result, that tabular output